client/reader/list-manage/tag-item.tsx

```diff
@@ -37,9 +80,27 @@ export default function TagItem( props: { item: Item; onRemove: ( e: MouseEvent
 					</div>
 				</a>
 			</div>
-			<Button primary onClick={ props.onRemove }>
-				{ translate( 'Remove' ) }
-			</Button>
-		</>
+
+			{ props.isFollowed && (
+				<FollowButton followLabel={ translate( 'Following site' ) } following />
+			) }
+
+			{ ! isInList ? (
+				<Button primary onClick={ addItem }>
+					{ translate( 'Add' ) }
+				</Button>
+			) : (
+				<Button primary onClick={ () => setShowDeleteConfirmation( true ) }>
+					{ translate( 'Remove' ) }
+				</Button>
+			) }
+
+			<ItemRemoveDialogue
```

Comment:
Nit: I think we use `Dialog` elsewhere in the codebase, even though this feels better to me (`en-gb`!).

Reply:
Hm, how did this happen? 😅